scripts/generate/build-rbac-prometheus-operator.sh

```diff
@@ -6,10 +6,12 @@ set -o pipefail
 # error on unset variables
 set -u
 
-rm -rf tmp
-mkdir tmp
-jsonnet -J scripts/generate/vendor scripts/generate/prometheus-operator-rbac.jsonnet > tmp/po.json
-jq -r 'keys[]' tmp/po.json | while read -r file
+TMP="tmp/gen"
+
+rm -rf "$TMP"
+mkdir "$TMP"
+jsonnet -J scripts/generate/vendor scripts/generate/prometheus-operator-rbac.jsonnet > "$TMP/po.json"
```

Comment:
Instead of storing the json data in a temporary file can't we use the same mechanism, we have in kube-prometheus? https://github.com/coreos/kube-prometheus/blob/master/build.sh#L18-L21

Reply:
I did it a bit differently to make it a bit clearer and easier to understand. Also no `xargs` as it isn't very portable.